lib/src/cmcd/CmcdFormatterOptions.ts

```diff
@@ -0,0 +1,29 @@
+import type { CmcdReportingMode } from './CmcdReportingMode.js';
+
+/**
+ * Options for formatting CMCD data values.
+ *
+ * @group CMCD
+ *
+ * @beta
+ */
+export type CmcdFormatterOptions = {
+	/**
+	 * The version of the CMCD specification to use.
+	 *
+	 * @defaultValue `1`
+	 */
+	version: number;
```

Comment:
Should it be optional?

```suggestion
	version?: number;
```